iset.mm

```diff
@@ -147339,6 +147339,448 @@ since the target of the homomorphism (operator ` O ` in our model) need
+      LZGALZMZHALZEALZMZUBZFGBNZHEBNZDNZVLVMCUCOZOZBNZFHDNZGEDNZBNZVKVLALZVMALZ
+      VNVQPVKCUDLZVEVFWAVDVGWCVJCUEQZVDVEVFVJUFZVDVEVFVJUGZABCFGIJUHRVKWCVHVIWB
+      WDVDVGVHVIUIZVDVGVHVIUJZABCHEIJUHRABCVODVLVMIJVOUKZKSTVKVLHVOOZEVOOZBNZBN
+      ZFWJBNZGWKBNZBNZVQVTVKCULLZVGWJALZWKALZWMWPPVDVGWQVJCUMQVDVGVJUNVKWCVHWRW
+      DWGACVOHIWIUPTVKWCVIWSWDWHACVOEIWIUPTABCWKFGWJIJUOUQVKVPWLVLBVKVDVHVIVPWL
+      PVDVGVJURWGWHABCVOHEIJWIUSRUTVKVRWNVSWOBVKVEVHVRWNPWEWGABCVODFHIJWIKSTVKV
+      FVIVSWOPWFWHABCVODGEIJWIKSTVAVBVC $.
+
+    $( Abelian group addition/subtraction law.  (Contributed by NM,
+       31-Mar-2014.) $)
+    abladdsub4 $p |- ( ( G e. Abel
+       /\ ( X e. B /\ Y e. B ) /\ ( Z e. B /\ W e. B ) )
+   -> ( ( X .+ Y ) = ( Z .+ W ) <-> ( X .- Z ) = ( W .- Y ) ) ) $=
+      ( wcel wa co wceq grpcl syl3anc ablsub4 syl122anc syl2anc cabl w3a ablgrp
+      wb 3ad2ant1 simp2l simp2r simp3l simp3r grpsubrcan syl13anc c0g cfv simp1
+      cgrp grpsubid oveq2d grpsubcl grprid 3eqtrd oveq1d grplid eqeq12d bitr3d
+      eqid ) CUALZFALZGALZMZHALZEALZMZUBZFGBNZHGBNZDNZHEBNZVODNZOZVNVQOZFHDNZEG
+      DNZOVMCUOLZVNALZVQALZVOALZVSVTUDVFVIWCVLCUCUEZVMWCVGVHWDWGVFVGVHVLUFZVFVG
+      VHVLUGZABCFGIJPQVMWCVJVKWEWGVFVIVJVKUHZVFVIVJVKUIZABCHEIJPQVMWCVJVHWFWGWJ
+      WIABCHGIJPQACDVNVQVOIKUJUKVMVPWAVRWBVMVPWAGGDNZBNZWACULUMZBNZWAVMVFVGVHVJ
+      VHVPWMOVFVIVLUNZWHWIWJWIABCDGFGHIJKRSVMWLWNWABVMWCVHWLWNOWGWIACDGWNIWNVEZ
+      KUPTUQVMWCWAALZWOWAOWGVMWCVGVJWRWGWHWJACDFHIKURQABCWAWNIJWQUSTUTVMVRHHDNZ
+      WBBNZWNWBBNZWBVMVFVJVKVJVHVRWTOWPWJWKWJWIABCDGHEHIJKRSVMWSWNWBBVMWCVJWSWN
+      OWGWJACDHWNIWQKUPTVAVMWCWBALZXAWBOWGVMWCVKVHXBWGWKWIACDEGIKURQABCWBWNIJWQ
+      VBTUTVCVD $.
+
+    $( Associative-type law for group subtraction and addition.  (Contributed
+       by NM, 19-Apr-2014.) $)
+    abladdsub $p |- ( ( G e. Abel /\ ( X e. B /\ Y e. B /\ Z e. B ) ) ->
+         ( ( X .+ Y ) .- Z ) = ( ( X .- Z ) .+ Y ) ) $=
+      ( cabl wcel w3a wa co wceq ablcom 3adant3r3 oveq1d syl3anc syl13anc simpl
+      cgrp ablgrp adantr simpr2 simpr1 simpr3 grpaddsubass grpsubcl 3eqtrd ) CK
+      LZEALZFALZGALZMZNZEFBOZGDOFEBOZGDOZFEGDOZBOZVAFBOZUQURUSGDULUMUNURUSPUOAB
+      CEFHIQRSUQCUCLZUNUMUOUTVBPULVDUPCUDUEZULUMUNUOUFZULUMUNUOUGZULUMUNUOUHZAB
+      CDFEGHIJUIUAUQULUNVAALZVBVCPULUPUBVFUQVDUMUOVIVEVGVHACDEGHJUJTABCFVAHIQTU
+      K $.
+
+    $( Cancellation law for subtraction.  (Contributed by NM, 2-Oct-2014.) $)
+    ablpncan2 $p |- ( ( G e. Abel /\ X e. B /\ Y e. B )
+       -> ( ( X .+ Y ) .- X ) = Y ) $=
+      ( cabl wcel w3a co c0g cfv wceq simp1 simp2 simp3 syl2anc syl13anc ablgrp
+      abladdsub cgrp syl eqid grpsubid oveq1d grplid 3eqtrd ) CJKZEAKZFAKZLZEFB
+      MEDMZEEDMZFBMZCNOZFBMZFUNUKULUMULUOUQPUKULUMQZUKULUMRZUKULUMSZVAABCDEFEGH
+      IUCUAUNUPURFBUNCUDKZULUPURPUNUKVCUTCUBUEZVAACDEURGURUFZIUGTUHUNVCUMUSFPVD
+      VBABCFURGHVEUITUJ $.
+
+    $( A cancellation law for commutative groups.  (Contributed by NM,
```

Comment:
"commutative groups" -> "Abelian groups"

Reply:
Since those are synonymous (both are, for example, mentioned in the opening sentence of https://en.wikipedia.org/wiki/Abelian_group ), I'm a bit reluctant to police which one is used.

But I guess we mostly use Abelian group so :shrug: I guess I can change this.